### ISSUE TITLE
Date Picker current date should have `aria-current="date"`

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#1978]
So:
`<button class="calendar-day grid-item selected" tabindex="0">22</button>`
should be:
`<button class="calendar-day grid-item selected" tabindex="0" aria-current="date">22</button>`

REF:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#date
